### DISCUSSION TITLE
A Nightly Build with 💥 Slicing Crash Analysis Tool 💥

### DISCUSSION THREAD

**Discussion by MariMakes** [Ultimaker/Cura#18080] [Releases]
We are seeing a significant increase in the amount of slicing crashes that are reported since Cura 5.x and are taking steps to analyze the crashes more systematically.  We'll need some help with that. 😇

<h3>We created a special 🌙 Nightly build 🌙 (5.7 Alpha)</h3>
This version of Cura has an updated slicing engine that will send a report to our Sentry tool when you encounter a slicing crash. Sentry will cluster the reports coming in, and point to where in the code the crash is likely coming from, so the Cura team can start working on resolving the biggest slicing crashes first. 
<br></br>

_This version has not passed any of our regular quality checks and we cannot guarantee it's reliability because it is still unfinished even though it may still be quite usable. For example, we are still working on fixing a crash when you open the printers settings or you might need to confirm that you trust the UltiMaker software. Before you upgrade to an upcoming 5.7 Beta or Stable, please delete this version including the configuration folder to prevent unwanted upgrading errors._ 

<h3>⬇️ Download Links </h3>

If you are using **Windows**, you can download a build here:
📁 [UltiMaker-Cura-5.7.0-alpha.0-win64-X64.exe](https://github.com/Ultimaker/Cura/releases/download/5.7.0-alpha.0/UltiMaker-Cura-5.7.0-alpha.0-win64-X64.exe)
📁 [UltiMaker-Cura-5.7.0-alpha.0-win64-X64.msi](https://github.com/Ultimaker/Cura/releases/download/5.7.0-alpha.0/UltiMaker-Cura-5.7.0-alpha.0-win64-X64.msi)

If you are using **Linux**, you can download a build here:
📁 [UltiMaker-Cura-5.7.0-alpha.0-linux-X64.AppImage](https://github.com/Ultimaker/Cura/releases/download/5.7.0-alpha.0/UltiMaker-Cura-5.7.0-alpha.0-linux-X64.AppImage)
📁 [UltiMaker-Cura-5.7.0-alpha.0-linux-X64.AppImage.asc](https://github.com/Ultimaker/Cura/releases/download/5.7.0-alpha.0/UltiMaker-Cura-5.7.0-alpha.0-linux-X64.AppImage.asc)

⚠️ There are no Mac builds available yet, we are working on resolving the build trouble we are encountering there. 

<h3>💡 Changes </h3>

Since this build is a work in progress, you can already benefit from fixes that are planned for the upcoming Cura release like:
- Faster slicing times for models with a lot of holes
- Improved Wall Ordering
- Decrease in floating support pieces
- A fix for a handful of crashes related to Tree Suppport
- A fix where extruder_nr template variables weren't added in start/end machine/extruder gcode 

**Please note**
Before reporting bugs on this build, please double-check if the issue is also present on the latest Stable Cura version. Right now that is [Cura 5.6.0](https://github.com/Ultimaker/Cura/releases/tag/5.6.0).  This Nightly Build looks differently on purpose so we can spot reports done with these builds.

<H3>Can you give it a try and let us know which issues are resolved? That would help a lot 😊</H3>

By default, we send the crash data anonymously. If you are comfortable with sharing more information and helping us narrow down where the different crashes are coming from, you can enable sharing your UltiMaker account username with us so we can link crashes to reports you have made in the past, or you plan to make in the future. You can sign in with your UltiMaker account in the top right corner and select the option to include your UltiMaker account name in your preferences.  

![image (164)](https://github.com/Ultimaker/Cura/assets/40423138/7b9672c5-55be-45da-ac7c-2ba6aa82c0fb)

**Comment by MariMakes** [comment]
@fieldOfView, @GregValiant, @Asterchades, @ThomasRahm, @5axes
We released a nightly that can help with the slicing crashes (and contains a fix for the slicing slowdowns). I'll update the slicing crash issue template to ask people with a crash to try this Cura 5.7 Alpha instead so we can see where the crash is coming from 🤞, feel free to send people this way if you feel that they could be helped with this nightly version, or use this build yourself if you are troubleshooting going forward.

If I have any updates on the data we are seeing coming in, I'll share them in this thread 👍

**Reply by admartin2001** [reply]
When I install Cura 5.7 Alpha and try to use it, I get a message that states that I do not have permission to use. I downloaded the file because when I installed a newer infill I kept getting the following error messages:
EnginePlugin
Couldn't start EnginePlugin: CuraEngineTiledinfill
No permission to execute process.

Slicing failed
Slicing failed with an unexpected error. Please consider reporting a  bug on our issue tracker.
I tried uninstalling Cura 5.6.0 and installed Cura 5.5.0. Still get the same error messages.
Now I don't seem to be able to slice anything no matter which version of Cura I use.

Went from printing everyday for a week to no printing at all now.
I would appreciate any help that you can provide.

**Reply by LilBub** [reply]
I am curious if your anti virus is blocking the execution of the program?  On mine, with Trend Micro anti virus, twice I have to allow Cura executables run.

**Comment by jellespijker** [comment]
If you are curious what kind of information is shared (when also allowing UltiMaker account name to be shared) this is a crashed on my computer.

https://ultimaker-o7.sentry.io/share/issue/90e7e22c71b043cb9c7ad041494cc392/

**Comment by jellespijker** [comment]
If you're curious what kind of crash data is logged as anonymous you can look at the following crash report 

https://ultimaker-o7.sentry.io/share/issue/96cdc8bbda8f47f7bd752377741ddbd1/

**Comment by GregValiant** [comment]
Thanks.
Starting a couple of days ago I was having a lot of issues with 5.6.0 and had to revert to 4.13.1 to get anything done.  Tree supports seemed to be the problem.

**Reply by arashiko28** [reply]
I am trying to slice very basic models, no supports involved and still crashes.

**Reply by dmpaul4691** [reply]
Same here. Tree supports cause errors, and when trying to save the project to submit the issue, it won't allow the file to save and just comes up with a generic "layer height" error message. Frustrating.

**Comment by GregValiant** [comment]
This model had errors but after fixing it (MS3D Builder) it still wouldn't slice #18072

**Comment by Sophist-UK** [comment]
I had a lot of intermittent slices whereby I would change a setting and it would automatically reslice and CRASH - but if I then resliced it without any further changes it would slice OK.

**Comment by Sophist-UK** [comment]
P.S. It's good that Sentry support is now added. I am just curious as to why it wasn't already included as standard?

**Reply by jellespijker** [reply]
We have it already in the Python front-end for a couple of years, but the Native C++ support was beta for a long time, at that time we tried to implement it but it proved to be unstable then.

We recently did an investigative spike with all our developers on how we can make our Engine more stable and found that support had been greatly improved also with a Conan package in the CCI (Conan Center Index), so it suddenly became a no-brainer to add it.

**Reply by Sophist-UK** [reply]
Thanks for the explanation. 😄

**Comment by sv-git-hub** [comment]
FYI, I'm using Cura 5.6.0, but I did have a slicing issue with an STL file from a downloaded design. When I used 5.7.0 as recommended with some slicing fixes, I did not have an issue with the slicing.

**Comment by Bibounet31** [comment]
hi! so its been a while now, that i can't manage to slice a file.. i tested a tons of stl, and nothing... this bug is present for me in every version exept the 5.5.0.. so im stuck in this one for now..
![image_2024-01-25_133614800](https://github.com/Ultimaker/Cura/assets/95425390/97dc5a86-a757-4bae-985a-67ac7602215b)

**Comment by PartySausage** [comment]
The Create Thumbnail Post Processor isn't working in this Alpha Version & generates the following log file entries. I originally thought there was a problem with a modified Post Processor that was based on this script that I'd created to work on my printer but I get the same error with the default unmodified version. Both versions work fine in 5.6

2024-01-25 15:48:16,953 - DEBUG - [MainThread] LocalFileOutputDevice.LocalFileOutputDevice.requestWrite [133]: Writing to [P:/Downloads/Print Data Test.gcode]...
2024-01-25 15:48:18,369 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.CreateThumbnail._createSnapshot [15]: Creating thumbnail image...
2024-01-25 15:48:18,372 - DEBUG - [MainThread] UM.Scene.Camera._preferencesChanged [255]: Changing perspective mode to perspective.
2024-01-25 15:48:18,377 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [61]: Loading [C:\Program Files\UltiMaker Cura 5.7.0-ALPHA\share\cura\resources\shaders\overhang.shader]...
2024-01-25 15:48:18,410 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [61]: Loading [C:\Program Files\UltiMaker Cura 5.7.0-ALPHA\share\cura\resources\shaders\transparent_object.shader]...
2024-01-25 15:48:18,431 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [61]: Loading [C:\Program Files\UltiMaker Cura 5.7.0-ALPHA\share\cura\resources\shaders\striped.shader]...
2024-01-25 15:48:18,520 - WARNING - [MainThread] cura.Snapshot.snapshot [188]: Exception: Failed to crop the snapshot!
2024-01-25 15:48:18,521 - WARNING - [MainThread] cura.Snapshot.snapshot [188]: Traceback (most recent call last):
2024-01-25 15:48:18,522 - WARNING - [MainThread] cura.Snapshot.snapshot [188]:   File "cura\Snapshot.py", line 186, in snapshot
2024-01-25 15:48:18,523 - WARNING - [MainThread] cura.Snapshot.snapshot [188]:     min_x, max_x, min_y, max_y = Snapshot.getImageBoundaries(pixel_output)
2024-01-25 15:48:18,523 - WARNING - [MainThread] cura.Snapshot.snapshot [188]:   File "cura\Snapshot.py", line 34, in getImageBoundaries
2024-01-25 15:48:18,524 - WARNING - [MainThread] cura.Snapshot.snapshot [188]:     min_y, min_x, min_a_ = numpy.amin(nonzero_pixels, axis=1)  # type: ignore
2024-01-25 15:48:18,525 - WARNING - [MainThread] cura.Snapshot.snapshot [188]:   File "<__array_function__ internals>", line 5, in amin
2024-01-25 15:48:18,526 - WARNING - [MainThread] cura.Snapshot.snapshot [188]:   File "numpy\core\fromnumeric.py", line 2879, in amin
2024-01-25 15:48:18,528 - WARNING - [MainThread] cura.Snapshot.snapshot [188]:   File "numpy\core\fromnumeric.py", line 86, in _wrapreduction
2024-01-25 15:48:18,530 - WARNING - [MainThread] cura.Snapshot.snapshot [188]: ValueError: zero-size array to reduction operation minimum which has no identity
2024-01-25 15:48:18,537 - DEBUG - [MainThread] LocalFileOutputDevice.LocalFileOutputDevice._performWrite [165]: Writing to Local File P:/Downloads/Print Data Test.gcode in text mode
2024-01-25 15:48:18,702 - DEBUG - [JobQueueWorker [2]] UM.FileHandler.WriteFileJob.run [80]: Writing file took 0.16207289695739746 seconds
2024-01-25 15:48:19,463 - INFO - [MainThread] SliceInfoPlugin.SliceInfo._onRequestFinished [315]: SliceInfo sent successfully

**Comment by BigD1950** [comment]
I installed the Nightly build as suggested above and fixed the Slicing failure, the print has many holed and using Tree support.....Thank you for the fix much appreciated
![Slicer 5 6 0 crashing and refusing to SliceSlicer ](https://github.com/Ultimaker/Cura/assets/59106570/f1538354-073a-4ef6-96a4-3ce6d14e3750)

**Comment by Slashee-the-Cow** [comment]
It's stalling when I try to slice even this simple model _with tree support turned on_ (and it doesn't need any support whatsoever, it's just part of my usual profile)
[E3V3S_PLA_corner_gluing_template_1_corner_high_nightly.zip](https://github.com/Ultimaker/Cura/files/14132608/E3V3S_PLA_corner_gluing_template_1_corner_high_nightly.zip)

**Reply by ThomasRahm** [reply]
Project file D:/Download/E3V3S_PLA_corner_gluing_template_1_corner_high_nightly.3mf contains an unknown machine type creality_ender3v3se_slashee. Cannot import the machine. Models will be imported instead.

I understand what you are trying to do, but this makes looking into bug reports harder than it should be...

You are correct though, this is another tree support related issue. Looking into it.

**Reply by ThomasRahm** [reply]
Crash occurs if there is no overhang to support.
Will do a pull request that prevents the crash, and also ensures that tree support generation is aborted early if there are no overhangs.

**Reply by Slashee-the-Cow** [reply]
D'oh! I meant to include this but obviously had a brain fart. It's an Ender-3 V3 SE with some changes to the gcode and setting the correct values for the print head size and gantry height.
[e3v3se slashee profile.zip](https://github.com/Ultimaker/Cura/files/14139994/e3v3se.slashee.profile.zip)

**Reply by ThomasRahm** [reply]
This pull request should fix this crash:
https://github.com/Ultimaker/CuraEngine/pull/2028

**Comment by BigD1950** [comment]
Than you, no issues now using the nightly upgrade…problem solved!

Desmond

**Comment by Sophist-UK** [comment]
Is there somewhere I can down load a more recent nightly build than the formal Alpha 0 release?

**Comment by 5axes** [comment]
in the Actions . Select one of the latest release :
https://github.com/Ultimaker/Cura/actions/workflows/installers.yml

You should find some alpha1 release.

**Reply by Sophist-UK** [reply]
Nightly windows builds are currently failing at the code signing stage - looks like it hangs and then times out or is manually cancelled. Workflow should probably set a reasonable timeout so that if it hangs it terminates much sooner. Last successful nightly build for Windows appears to have been 26 Jan.

But I found a successful Alpha1 Windows build from 26 Jan. I just need to see what was included in that.

**Reply by Sophist-UK** [reply]
Ah - unfortunately the 25 Jan build was too long ago and the artefacts have expired.

So, no, there are no newer 5.7 Alpha builds than the one in the Alpha0 release.

I guess I have to wait for the code signing to be fixed.

**Reply by Sophist-UK** [reply]
P.S. @5axes As an aside, just about to use your Tab-AntiWarping plugin for the first time as a brim is not preventing bed separation with a new PLA filament I am using.

**Comment by Fylhtq12** [comment]
For some reason, since this version (5.6.0), the dimensions have become smaller. Previously, I used to print with any plastic - all sizes corresponded to the drawing. Now I'm printing - the dimensions of the steel are 0.5 mm, or even 1 mm smaller.

**Comment by seacoboy** [comment]
I have tried a few times to update my version 5.6 to 5.7 but windows will not allow me to install it, it thinks it's malware. I have tried running as administrator but no joy, any ideas how I can bypass this problem?
 
<a href="https://vgy.me/u/91Misc"><img src="https://i.vgy.me/91Misc.jpg" alt="91Misc.jpg"></a>

**Reply by Sophist-UK** [reply]
That pop-up is more likely an indication of an unsigned executable rather than malware.

And providing you have administrator access to your PC you should be able to enable it to be installed. (You will need to research it.)

**Reply by seacoboy** [reply]
Thanks for the advice, I went into regedit and disabled UAC.

**Reply by jellespijker** [reply]
> Thanks for the advice, I went into regedit and disabled UAC.

Please, don't disable UAC or lower the level. It's there for a reason.

If you're seeing this pop-up it's likely caused due to us having some signing certificate issues at the moment. We're in contact with our Certificate Authority issuer to fix this asap.

Either wait a couple of days till this is resolved (only an issue for this Alpha) or click the "more info" button on the dialog that popped up and then click on "run anyway"

**Reply by Sophist-UK** [reply]
I agree with @jellespijker.

@seacoboy, I did NOT suggest that you should disable UAC - this is a VERY bad idea indeed.

**Reply by seacoboy** [reply]
Sorry should have added, I switched UAC back on directly after installing Cura.

**Reply by jellespijker** [reply]
Windows installer have been re-uploaded and are signed with a new certificate. If a dialog pops-up you should be able to click more info. This is because ~1000 people need to trust this new certificate. After that the dialog shouldn't pop-up anymore.

**Comment by GregValiant** [comment]
#18271 is being difficult.
Opening the 3mf as a project in 5.7 caused an immediate crash.  It opens in 5.6.0 alright.
Bringing the model into 5.7 it will slice at 0.2 layer height by not at 0.16.  The slice doesn't look right.  Looks like a detached retina or cornea or something.
![image](https://github.com/Ultimaker/Cura/assets/64202104/a04784c1-558c-43a5-9e86-8c299b6d95b1)

This is the slice in 5.6.0.  It will slice at 0.16 but not at 0.20
![image](https://github.com/Ultimaker/Cura/assets/64202104/c864d539-8b17-4b72-8466-46cd48ccbc85)

**Comment by Sophist-UK** [comment]
Possible memory leak in Cura 5.6.

With two copies of Cura loaded, one with a single smallish part and the other with a relatively simple Retraction Test object loaded, I see the following message (the latest before a recent reboot) in my Windows Event Log:

> Windows successfully diagnosed a low virtual memory condition. The following programs consumed the most virtual memory: UltiMaker-Cura.exe (44460) consumed 7058538496 bytes, UltiMaker-Cura.exe (42892) consumed 5970608128 bytes, and Messenger.exe (17764) consumed 3430383616 bytes.

In more readable language the two instances are consuming 7GB and 6GB of memory!!!

To put this in context, when I load first Cura it uses c. 500MB of memory, but as I watch an idle instance's memory usage, even though it is idle and minimised it still does a significant amount of I/O (c. 10k I/Os of some sort per second), and the memory usage grows by 30k per second. 

Here is a a previous one from two days previously (same instances):

> Windows successfully diagnosed a low virtual memory condition. The following programs consumed the most virtual memory: UltiMaker-Cura.exe (44460) consumed 6267150336 bytes, UltiMaker-Cura.exe (42892) consumed 5178519552 bytes, and Messenger.exe (17764) consumed 3249709056 bytes.

Am not sure which instance of Cura was which, but one of these had been idle for this entire period - literally no clicks. But memory of both instances has grown.

**Reply by ThomasRahm** [reply]
Do you have a project file to reproduce? 
I reported an issue https://github.com/Ultimaker/Cura/issues/14992 some time ago that can seem like a memory leak, even though it is "just" a regular bug

**Reply by Sophist-UK** [reply]
I am seeing I/Os and memory increasing on a copy of Cura that is opened without any model loaded.

I just loaded Cura without any plugins and far fewer I/Os (but still some), but still memory usage growth about the same.

**Reply by Sophist-UK** [reply]
I opened Cura at mid-day yesterday and it was using c. 450MB of memory, and after c. 21 hours this has grown to 2GB. So this memory leak is c. 74MB per hour.

**Reply by HellAholic** [reply]
I've added this to an existing ticket for further investigation.
For internal reference CURA-8372

**Comment by DragonLord1981** [comment]
I cant even instal this program.. not evan as a adminstarot windows just BLOCK it

**Reply by Sophist-UK** [reply]
I think we need to wait until they fix their code signing certificate issue. Once that has been done, we should get a version we can install.

**Reply by jellespijker** [reply]
@saumyaj3 can you let them know in this thread once the certificates are fixed again?

**Reply by jellespijker** [reply]
Windows installer have been re-uploaded and are signed with a new certificate. If a dialog pops-up you should be able to click more info. This is because ~1000 people need to trust this new certificate. After that the dialog shouldn't pop-up anymore.

**Comment by Skipper488** [comment]
I get an error telling me an administrator has blocked me from running this app when trying to install 5.7.0 a. Only problem is I'm the administrator and only user of this computer. Even tried running as administrator.

**Reply by Sophist-UK** [reply]
See above. The executable code signing in their Github actions is broken so the executables are unsigned. You should be able to click See More (or something similar) and then install.

**Reply by Asterchades** [reply]
Different error to the one you're thinking of. This is quite literally a box with only two possible actions: closing it via the X in the corner, or closing it via the "Close" button.

The only way to circumvent this one (without opening a gaping security hole) is to run it from an elevated command prompt.

**Reply by Sophist-UK** [reply]
Ah - well you are probably safe enough running the installer from an elevated command prompt given the source.

Or you do as I do and wait for Ultimaker to treat this with the sense of urgency it deserves and fix the code signing.

**Reply by jellespijker** [reply]
We are aware of the issue, which only occurs with this Alpha version. We have been in contact with the certificate issuer on a daily basis and doing the best we can to get the new Dongle needed for codesigning on the location of the GH runner.

I do have some choice words to say about how the Certificate issuer is picking this up. That being said the problem occurs on an Nightly version so the damage is minimal.

> [!NOTE] 
> The issuer has finally send a new dongle and we expect to have it on location on Monday

**Reply by Sophist-UK** [reply]
Hurrah!!

Nice to know that Ultimaker **is** treating it with a sense of urgency and it is the certificate issuer that isn't.

As and when we have a signed version, I will give it a test.

**Reply by jellespijker** [reply]
Windows installer have been re-uploaded and are signed with a new certificate. If a dialog pops-up you should be able to click more info. This is because ~1000 people need to trust this new certificate. After that the dialog shouldn't pop-up anymore.

**Comment by jellespijker** [comment]
> [!IMPORTANT]
> Windows installer have been re-uploaded and are signed with a new certificate. If a dialog pops-up you should be able to click more info. This is because ~1000 people need to trust this new certificate. After that the dialog shouldn't pop-up anymore.

**Comment by Flight** [comment]
Would be really nice to have a Mac OS arm64 version compiled, as the latest Mac OS build already expired =( Or maybe someone has is saved and can share it?

**Reply by jellespijker** [reply]
macOS is broken for 5.7.0-alpha.0 it is fixed on main so it will be 5.7.0-beta.0 you have to be patient for a little while longer tho

**Comment by StephanEncee** [comment]
Is it possible with the 5.7.0 stable version to connect direcly the Method X and Method XL?

**Reply by jellespijker** [reply]
Not over LAN, but if you have digital Factory account you can send your sliced makerbot files to the Method X and XL via the cloud connection in 5.7

**Comment by ConstantChange** [comment]
Had a crash using 5.6.0 Stable. 

Same settings as for several prior models. Then, no go.

**Comment by jonashogstrom** [comment]
Had a failure to slice the following model in 5.6.0: https://www.printables.com/model/616628-heltec-wireless-tracker-case-for-meshtastic/files#preview:file-e9aSl
5.7.0-alpha sliced the same file just fine. Gracias!

**Comment by GregValiant** [comment]
#18463 sliced after changing the Initial Layer Height from 0.28 to 0.20.
Error code is 3221225477

**Comment by RealDanCarroll** [comment]
I dont know if you guys are aware.... "Nightly Builds" should never become MONTHS old....

Please update this horribly unstable software. Im trying NOT to switch to a completely different alternative.

**Reply by PartySausage** [reply]
This was just one particular nightly build. The most recent ones can be found under the actions tab

**Reply by Sophist-UK** [reply]
@gh13 @RealDanCarroll 

As far as I can tell from the GH workflow https://github.com/Ultimaker/Cura/blob/main/.github/workflows/installers.yml it successfully did:

```
 gh release upload nightly-5.6 installers/UltiMaker-Cura-5.6.1-beta.0-win64-X64.exe --clobber
```

However, there is no release with a tag `nightly-5.6` and so this presumably goes in the bitbucket rather than into a directory `/installers`.

**Reply by jellespijker** [reply]
The nightlies get published in a draft release for QA.

We are discussing releasing them to the public. But there are some issue to resolve before we provide access to a bigger audience.

Such as upgrade paths between versions and making it more recognizable that they are nightlies and not intended for production use.

**Comment by olmosbananas** [comment]
"A fix where extruder_nr template variables weren't added in start/end machine/extruder gcode"

this seemed to fix the outputted gcode errors i was previously getting, but still does not properly let me select my 2nd extruder out of cura -- it has no effect, always print on extuder 1 no matter what I do.

only way to get it to print on my 2nd extruder is to do a "find and replace" in the gcode and changing T0 into T1.

**Comment by suttonb** [comment]
I love this feature! A couple suggestions:

- Giving the option to add a message with the crash report could be helpful. E.g. I have a case where it's crashing only when I turn Adaptive Layers off, but when I turn that setting back on it slices fine. Being able to note that in the report might help speed up debugging.
- Giving the option not to send a crash report would be nice. E.g. if I know I've already submitted a report for a given crash and I hit the slice button again and it crashes a second time, I could choose not to send a report that I know would be a duplicate of the first one.

**Comment by Sophist-UK** [comment]
Cura 5, simple open-top calibration cube, 15mm across, 0.5mm line width, zig-zag bottom pattern. Layer 2:

![image](https://github.com/Ultimaker/Cura/assets/3001893/0feca578-8731-460d-bd7a-00ce9168dee7)

Why are there separate disconnected lines at top and bottom rather than a straight forward zigzag?

**Comment by LilBub** [comment]
Just curious if a recent Windows 11 Update has caused the issues?  I was having good success until an update about 2 weeks ago.

**Reply by Sophist-UK** [reply]
I would imagine that this would depend on exactly what specific issues you are having.

**Reply by LilBub** [reply]
Slicing fails 90% of the time

**Reply by suttonb** [reply]
You could try rolling back the updates to see if that fixes things, although it sounds like you may only have about 10 days before it clears the update cache. This link has the steps: https://www.dell.com/support/kbdoc/en-us/000196921/how-to-roll-back-a-windows-11-update-if-there-are-issues

**Reply by LilBub** [reply]
I have other applications (video editing, SolidWorks, others...) that are working fine, so I don't think I would want to roll it back.  Sadly, when Cura implodes repeatedly, I resort to another slicer

**Comment by admartin2001** [comment]
Thank you for your quick response. I don't think my Anti Virus program is blocking the software. I use PC Matic anti virus software and have not had any issues with it so far. What I did for now is I downloaded PURSAslicer to do my slicing. I'm hoping to get back to using CURA because I have been using it for a long time and am use to it.

________________________________

**Reply by LilBub** [reply]
Try the new Alpha that just came out

**Comment by admartin2001** [comment]
Well I finally figured it out. All I had to do is remove (delete) the Tile
Infill Plugin that I had downloaded just prior to having the problems with
slicing. Now I can slice without any trouble but without that particular
plugin.
I'm a happy man now and will not try new plugins for a while.
Thanks for your help.

P.S. I tried the new Alpha version of cura but it said I didn't have
permission to use it. One day I'll figure it out, but now I am good with
the non-alpha version. I never understood why I don't have full control
(permission) over my own personal home computer where I am the only one
using it.

**Comment by jjonas20088** [comment]
ola, estou com problema de inicialização do cura , não abre de jeito nenhum, fica na tela azul carregando....
